newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md

```diff
@@ -0,0 +1,147 @@
+An intense discussion about the strategy to be used here, and how much specific HEP solutions
+are really required, was had and typifies many of the open questions
+our R&D programme needs to address. 
+
+<!-- TrickTrack, ACTS mention? -->
+
+For producing simulated events there were two full sessions. 
+Producing events with better underlying physics is needed for both
+HL-LHC and the intensity frontier programs, where accuracy of understanding
+rare processes is a key to success. In addition, there is a need to speed
+up simulation. This takes a few forms: better use of current hardware,
+tested via the GeantV R&D line and the use of approximate techniques that reproduce
+complex processes more cheaply. Machine learning again looks very promising
+here, particularly for calorimiter simulation. Past fast simulation efforts
+have been very detector specific so the challenge is how to make faster
+techniques more generic, including the process of training networks and
+performing validation. There is a very active community here and a vigorous
+and diverse R&D is underway.
+
+The Software Development session reviewed progress made by the
+HSF's [Packaging Working Group](/activities/packaging.html){:target="_naples_packaging_wg"}. A common
+solution here will be of great benefit to our R&D projects in the future
+as it will allow a wider base of users to more easily test early
+releases. A decision was also made to consolidate information and best
+practice on performance monitoring and tooling, which will help developers
+across the field.
+
+Helping establish best practice for development is one part of the training 
+challenge our community faces. Ensuring that our software is well written
+and tested is a key part of making code more sustainable for experiments
+that now have multi-decade lifetimes. The training session gathered speakers
+involved in may of the communitiy's training areas, in experiments and
+from dedicated schools, and discussion about the best way to address the
+needs in the future was had. Agreeing that a solid base was the best place
+to start, the group will try and ensure that as many basic training needs
+are satisfied in common. Contributions to training should be recognised
+as a part of the career profile of our developers, but long term career
+prospects remain a concern, as with many other areas in HEP.
+
+<div style="text-align:center; padding:25px"><img src ="/images/naples-ws-banquet.jpg" alt="Well fed at the banquet" /></div>
+
+Opening the final session of the workshop was 
+[a talk from Anna Scaife](https://indico.cern.ch/event/658060/contributions/2940455/attachments/1625101/2587580/WLCG-180329.pdf){:target="_naples_ska"}
+from the Square Kilomoeter Array Telescope and AENEAS projects. This is
+one of the other sciences facing a data deluge in the future and there
+can be a very healthy relationship between HEP, which has been recognised
```

Comment:
Maybe rephrase a bit since one would expect "between A and B" but there's no B. Maybe just between -> with will do.